wagtail/wagtailcore/models.py

```diff
@@ -351,6 +351,7 @@ class Page(six.with_metaclass(PageBase, AbstractPage, index.Indexed, Clusterable
 
     search_fields = [
         index.SearchField('title', partial_match=True, boost=2),
+        index.FilterField('title'),
```

Comment:
LGTM, but again could do with a test (even if it's just confirming that `Page.objects.order_by('title').search('blah', order_by_relevance=False)` completes without an error)

Reply:
Test for searching an ordered queryset added in 6bdf75c2aa2f7629be9a9b4f479208236d3d8c61. Not entirely what I was going for, but it looks like we don't have a good place to put tests that check that `Page`'s search configuration does what we want it to, and creating such a place (and setting it up to exercise all search backends) is going to need more heavy refactoring of our test suite than we can comfortably do here.

Reply:
I've now come up with a good way of testing search backends against the Page model, as part of my fix for #4015 - but that's just revealed that `Page.objects.order_by('title').search('blah', order_by_relevance=False)` is _also_ broken on the Postgres backend. Argh!

Reply:
...ah, that's #3726, which I guess we ought to backport to 1.12 along with the other fixes. Sorry for the noise :-)